### ISSUE TITLE
KaTeX plugin malfunctioning in TiddlyDesktop Browser

### ISSUE THREAD

**Issue by kramsretlow** [Jermolene/TiddlyDesktop#124]
I am trying out TiddlyDesktop 0.0.9, using wikifolders I've created using the node.js `tiddlywiki mynewwiki --init server` method.  I am very happy with the setup, but when I try to use math markup using the KaTeX plugin, I get the message `TypeError: Failed to execute 'appendChild' on 'Node': parameter 1 is not of type 'Node'` wherever I've used the $$ $$ markup.  

This happens in the browser that's launched when I click "open" in TiddlyDesktop.  

On the other hand, if I click "advanced" and set a host and port, and then navigate to that host:port in Firefox or MS Edge, the math is rendered correctly.

**Comment by Jermolene** [comment]
Hi @kramsretlow thanks for the report, and apologies for the delay in responding.

I had a couple of questions:

* What operating system are you using?
* Which is the default browser that opens which you click "open" in TiddlyDesktop?
* How did you install the KaTeX plugin? Confusingly, the correct way is to edit the tiddlywiki.info file in your wiki folder, and not to install it via the plugin library in control panel

**Comment by kramsretlow** [comment]
Thanks for following up.  Here are the answers to your questions:

* I'm using Windows 10 Home, version 1709
* When I click "open", the wiki opens in the nwjs Chromium browser.  At least that's what I assume it is.  The browser has no menu bar or settings button, and when I right-click and choose "inspect" I get a chrome developer console.  I don't have Chrome installed on the machine.  Browser processes appear as "nwjs" in Task Manager.  (Incidentally, these tasks don't terminate when I close TiddlyDesktop, but perhaps that's a topic for another day...)
* Initially I had tried the drag-and-drop method.  After seeing the documentation about it I removed the plugin and did the tiddlywiki.info method.  Just now I tested out a fresh wiki using the proper installation method and the problem remains.
